File: Domain/Persona.cs
using System;

namespace Domain
{
    public class Persona
    {
        public Guid Id { get; set; }
        public string Nombre { get; set; }
        public string Apellido { get; set; }
        public string Dni { get; set; }
        public string Telefono { get; set; }
        public string Celular { get; set; }
        public string Email { get; set; }
        public string Cuil { get; set; }
        public DateTime FechaNacimiento { get; set; }
    }
}

File: client-app/src/app/models/Persona.ts
export interface IPersona {
  id: string;
  nombre: string;
  apellido: string;
  dni: string;
  telefono: string;
  celular: string;
  email: string;
  cuil: string;
  fechaNacimiento: Date;
}
export interface IPersonaFormValues extends Partial<IPersona> {
  time?: Date;
}
export class personaFormValue implements IPersonaFormValues {
  id?: string = undefined;
  nombre: string = "";
  apellido: string = "";
  dni: string = "";
  telefono: string = "";
  celular: string = "";
  email: string = "";
  cuil: string = "";
  fechaNacimiento?: Date = undefined;
  time?: Date = undefined;

  constructor(init?: IPersonaFormValues) {
    if (init && init.fechaNacimiento) {
      init.time = init.fechaNacimiento;
    }
    Object.assign(this, init);
  }
}
